cernopendata/modules/fixtures/data/docs/cod-portal-upgrade-31-1-2024/cod-portal-upgrade-31-1-2024.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "author": "CERN",
+    "body": {
+      "content": "cod-portal-upgrade-31-1-2024.md",
+      "format": "md"
+    },
+    "collections": [
+      {
+        "primary": "News"
+      }
+    ],
+    "date_published": "2024-01-31",
+    "featured": 1,
+    "short_description": {
+      "content": "New version of the CERN Open Data Portal deployed"
+    },
+    "slug": "cod-portal-upgrade-31-1-2024",
```

Comment:
We are using YYYY-MM-DD everywhere, so if you want to use this slug, better say `cod-portal-upgrade-20240131`.

Reply:
Done, thanks